src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepReviewsResource.kt

```diff
@@ -22,55 +22,15 @@ import uk.gov.justice.digital.hmpps.incentivesapi.dto.CurrentIepLevel
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.IepDetail
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.IepReview
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.IepSummary
-import uk.gov.justice.digital.hmpps.incentivesapi.dto.prisonapi.IepLevel
-import uk.gov.justice.digital.hmpps.incentivesapi.service.IepLevelService
 import uk.gov.justice.digital.hmpps.incentivesapi.service.PrisonerIepLevelReviewService
 import uk.gov.justice.digital.hmpps.incentivesapi.util.ensure
 
 @RestController
 @RequestMapping("/iep", produces = [MediaType.APPLICATION_JSON_VALUE])
 @Tag(name = "Incentive reviews", description = "Retrieve and add incentive review records. Ported from prison-api")
-class IepLevelResource(
-  private val iepLevelService: IepLevelService,
+class IepReviewsResource(
```

Comment:
not too bothered, but maybe `IncentiveReviewsResource`? or something else that doesn't use the legacy term IEP?

Reply:
I deliberately kept the "iep" prefix for 2 reasons:
1. it's predictable and less confusing given the endpoints' URLs start with `/iep/reviews/` (and `/iep/levels`)
2. these endpoints are still using kinda legacy DTO/classes and we may want to rethink them (e.g. create new ones and deprecate these)

That's why I didn't want to create even more confusion by renaming this to something that may suggest these are "new" endpoints if that makes sense :)

Reply:
👌

Reply:
(I'm also thinking these are really Incentive records given they may not necessarily be Incentive "reviews", but that's another story ☺️)

Reply:
that's fair but IEP still boo ;)

Reply:
Agreed, IEP boo :)